test/e2e/tinkerbell_test.go

```diff
@@ -1516,6 +1516,61 @@ func TestTinkerbellKubernetes127UbuntuSimpleFlow(t *testing.T) {
 	runTinkerbellSimpleFlow(test)
 }
 
+func TestTinkerbellKubernetes123Ubuntu2204SimpleFlow(t *testing.T) {
```

Comment:
Is the idea to convert these into upgrade tests at some point? I feel like that probably adds better coverage with no a lot of overhead.

Reply:
Yeah I plan to add upgrade tests of the following format:
* `1.26 Ubuntu 2004` to `1.26 Ubuntu 2204`
* `1.26 Ubuntu 2204` to `1.27 Ubuntu 2204`

Reply:
ok cool
When you do that, can we remove these simple flow ones? They will probably be redundant

Reply:
Cool, will do.